sdk/src/store.rs

```diff
@@ -1903,6 +1899,24 @@ impl Store {
         }
     }
 
+    /// load a CAI store from  a file
+    ///
+    /// in_path -  path to source file
+    /// validation_log - optional vec to contain addition info about the asset
+    #[cfg(feature = "file_io")]
+    fn load_cai_from_file(
+        in_path: &Path,
+        validation_log: &mut impl StatusTracker,
+    ) -> Result<Store> {
+        match Self::load_jumbf_from_path(in_path) {
+            Ok(manifest_bytes) => {
+                // load and validate with CAI toolkit
+                Store::from_jumbf(&manifest_bytes, validation_log)
+            }
+            Err(e) => Err(e),
```

Comment:
The error may not be meaningful if you don't pass a StatusTracker to load_jumbf_path.  You may just get LogStop.

Reply:
This was taken from the previous code that called load_jumbf_from_File(), which also does not take a StatusTracker.
load_cai_from_file had these three error return states that do log log to a StatusTracker:
                            // return an error with the url that should be read
                            Err(Error::RemoteManifestUrl(ext_ref))
                        }
                    } else {
                        Err(Error::JumbfNotFound)
                    }
                }
            }
            Err(e) => Err(e),

Reply:
So this should create the same result.

Reply:
Maybe I am misreading this.